_posts/patterns/2016-01-28-alerts.md

```diff
@@ -0,0 +1,275 @@
+---
+layout:         pattern
+title:          Alerts
+date:           2015-01-28 00:00:00
+
+categories:     patterns
+tags:
+- atomic
+- pattern
+- alerts
+
+slug:           alerts
+
+url_styles:     "patterns/_alerts.scss"
+
+description:    Examples of alerts in a semi-real world context.
+
+info:           Alerts provide information about...TBD
```

Comment:
@chris-mike -- this text is what is displayed in the PL documentation site.  Please let me know what this can be updated to.  E.g. [UXPL - Button](http://ux.edx.org/#buttons).